zen_queries/tests/tests.py

```diff
@@ -106,6 +107,17 @@ def test_template_response(self):
             response.render()
 
 
+class TemplateTagTestCase(TestCase):
+    def test_queries_disabled_template_tag(self):
+        widgets = Widget.objects.all()
+        with self.assertRaises(QueriesDisabledError):
+            django_render(None, "queries_disabled_tag.html", {"widgets": widgets})
+
+    def test_queries_dangerously_enabled_template_tag(self):
+        widgets = Widget.objects.all()
+        render(None, "queries_dangerously_enabled_tag.html", {"widgets": widgets})
```

Comment:
you could bring the example/test templates closer to the tests with something like:
```
from django.template import Template, Context
template_to_render = Template(
   '{% load zen_queries %}'
   '{% queries_disabled %}',
   '{{ widgets.count }}',
   '{% end_queries_disabled %}'
)
rendered_template = template_to_render.render(Context({"widgets": widgets}))
```

Reply:
I kinda think I like the version with the separate templates better to be honest. And it more closely matches what's happening in a real project.